cli/command/image/build_linux_test.go

```diff
@@ -0,0 +1,56 @@
+//+build linux
+package image
+
+import (
+	"bytes"
+	"io"
+	"io/ioutil"
+	"runtime"
+	"syscall"
+	"testing"
+
+	"github.com/docker/cli/internal/test"
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/pkg/archive"
+	"github.com/gotestyourself/gotestyourself/fs"
+	"github.com/gotestyourself/gotestyourself/skip"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/net/context"
+)
+
+func TestRunBuildResetsUidAndGidInContext(t *testing.T) {
+	skip.IfCondition(t, runtime.GOOS == "windows", "uid and gid not relevant on windows")
```

Comment:
This check is actually not relevant as `assert.Equal(t, uint32(0), fileInfo.Sys().(*syscall.Stat_t).Uid)` doesn't compile under windows… I should remove that `skip` completely.

Reply:
Isn't this file already skipped on Windows?